adobecom/milo
# MWPW-147989 and MWPW-151786: handle localized path for quiz spreadsheets and doc files

* 1. add localized prefix for all paths in quiz and quiz results spreadsheets if need, including quiz, quiz-results, footer fragments
* 2. add localized prefix for all paths in quiz and quiz results dic files if need, including quiz, quiz-results
* 3. add unit tests
* 4. so the logic is if you already add localized prefix, the code will skip it, if not and not en-US, the code will add localized prefix automatically

Resolves: [MWPW-147989](https://jira.corp.adobe.com/browse/MWPW-147989)
[MWPW-151786](https://jira.corp.adobe.com/browse/MWPW-151786)

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/?martech=off
- After: https://stage--milo--JackySun9.hlx.page/?martech=off

How to test?
Test with ES, it is using all paths without country code:
URL: https://stage--cc--adobecom.hlx.page/es/creativecloud/plan-recommender/quiz2?milolibs=stage--milo--JackySun9
You can see https://stage--cc--adobecom.hlx.page/es/creativecloud/plan-recommender/quiz2 is loading English content,
![Screenshot 2024-06-04 at 11 19 54 AM](https://github.com/adobecom/milo/assets/67350368/cc1d3826-85e0-441d-ad7d-59e5ae34bbc2)

but with updated code it is loading ES content, also we set all fragment paths in LocalStorage
![Screenshot 2024-06-04 at 11 19 46 AM](https://github.com/adobecom/milo/assets/67350368/bea1d7a9-7509-4be0-9bea-77df597ccf2d)
![Screenshot 2024-06-04 at 11 19 29 AM](https://github.com/adobecom/milo/assets/67350368/cb58ed38-f731-4dfd-9b8d-fcc98b10be71)


Test with DE, it is using all paths with country code:
URL: https://stage--cc--adobecom.hlx.page/de/creativecloud/plan-recommender/quiz2?milolibs=stage--milo--JackySun9
